src/Utilities/Compiler/Options/SymbolNamesWithValueOption.cs

```diff
@@ -224,7 +224,7 @@ public bool Contains(ISymbol symbol)
         /// <summary>
         /// Gets the value associated with the specified symbol in the option specification.
         /// </summary>
-        public bool TryGetValue(ISymbol symbol, [MaybeNullWhen(false)] out TValue value)
+        public bool TryGetValue(ISymbol symbol, [MaybeNullWhen(false)] out TValue? value)
```

Comment:
❕ The attribute here no longer makes sense.

```suggestion
        public bool TryGetValue(ISymbol symbol, out TValue? value)
```